Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,199 @@
+
+#. `<Topic> Guide`, e.g. "Frontend Localization Guide",
+   for collections of articles on a specific topic
+#. `<Topic> Reference`, e.g. "TSconfig Reference",
+   for a complete reference
+#. `<Topic> Tutorial`, e.g. "Editors Tutorial",
+   for collections of tutorials on a specific topic
+
+.. _settings-guides-version-and-release:
+
+Version and release
+-------------------
+
+The properties *version* and *release* both contain the version of the manual
+and mostly correspond to the version of the TYPO3 LTS or TYPO3 extension to
+which the documentation refers.
+
+The version is shown below the title in the theme's release switch and in the
+title meta tag, the release is not shown currently  - but it should be
+kept anyway to satisfy internal requirements.
+
+version
+    The major project version, used as the replacement for \|version\|.
+    For example this may be something like 12.4.
+
+release
+    The full project version, used as the replacement for \|release\|.
+    For example 13.0.0-dev.
+
+If you do not need the separation provided between version and release,
+just set them both to the same value.
+
+.. _settings-guides-copyright:
+
+Copyright
+---------
+
+The *copyright* property contains the copyright claim of the project. It is
+displayed in the footer as "© Copyright <copyright>" and has in most use cases
+of the TYPO3 world the values:
+
+#.  `since <creation-year> by the TYPO3 contributors`,
+    for example "since 1999 by the TYPO3 contributors" (official TYPO3 manuals and TYPO3
+    system extensions)
+#.  `since <creation-year> by <vendor> & contributors`,
+    for example "since 1999 by dkd & contributors" (third-party TYPO3 extensions)
+
+.. _settings-guides-theme:
+
+TYPO3 Theme settings
+====================
+
+Further settings can be made in the XML tag
+:xml:`<extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension">`.
+The class attribute is mandatory, it references the extension that is used
+to render the documentation with the TYPO3 documentation theme.
+
+.. _settings-guides-github-workflow:
+
+Configure the "Edit on GitHub" button
+-------------------------------------
+
+The *github_repository* and *github_branch* properties must be set to provide
+the "Edit on GitHub" button, which allows the reader to jump from the currently
+viewed documentation directly to the associated GitHub repository and edit the
+page file.
+
+If the project is hosted on GitHub and public contributions are desired, these
+properties should be set accordingly:
+
+1.      `<user>/<repository>`, for example to "TYPO3-Documentation/TYPO3CMS-Reference-TCA"
+        or "FriendsOfTYPO3/extension_builder".
+2.      `<branch>`, for example on "main" or "10.x".
+
+For example:
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml, excerpt
+
+    <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+               edit-on-github-branch="main"
```

Comment:
Why that much whitespace before the attribute?

Reply:
to allign all atributes. My IDE does that automatically, yours not?